packages/contracts-bedrock/test/CommonTest.t.sol

```diff
@@ -31,46 +37,186 @@ import { AddressManager } from "src/legacy/AddressManager.sol";
 import { L1ChugSplashProxy } from "src/legacy/L1ChugSplashProxy.sol";
 import { IL1ChugSplashDeployer } from "src/legacy/L1ChugSplashProxy.sol";
 import { CrossDomainMessenger } from "src/universal/CrossDomainMessenger.sol";
+import { GovernanceToken } from "src/governance/GovernanceToken.sol";
 import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
 import { LegacyMintableERC20 } from "src/legacy/LegacyMintableERC20.sol";
+import { LegacyMessagePasser } from "src/legacy/LegacyMessagePasser.sol";
 import { SystemConfig } from "src/L1/SystemConfig.sol";
 import { ResourceMetering } from "src/L1/ResourceMetering.sol";
 import { Constants } from "src/libraries/Constants.sol";
+import { Deploy } from "scripts/Deploy.s.sol";
 
-contract CommonTest is Test {
+contract CommonTest is Deploy, Test {
     address alice = address(128);
     address bob = address(256);
-    address multisig = address(512);
 
-    address immutable ZERO_ADDRESS = address(0);
-    address immutable NON_ZERO_ADDRESS = address(1);
-    uint256 immutable NON_ZERO_VALUE = 100;
-    uint256 immutable ZERO_VALUE = 0;
-    uint64 immutable NON_ZERO_GASLIMIT = 50000;
-    bytes32 nonZeroHash = keccak256(abi.encode("NON_ZERO"));
-    bytes NON_ZERO_DATA = hex"0000111122223333444455556666777788889999aaaabbbbccccddddeeeeffff0000";
```

Comment:
I don't love these values, especially because they were not standardized on. OTOH, seeing literal values in the tests ([eg](https://github.com/ethereum-optimism/optimism/blob/feat/ctb-test-deploy-2/packages/contracts-bedrock/test/BenchmarkTest.t.sol#L87)) feels a bit like a magic value, ie. it's unclear if it matters that the call is to `address(1)`. 

I liked how when we were using ethers we'd use `constant.ZeroAddress`, perhaps we could create a `library TestConstants` and use a similar syntax for these kinds of dummy values?

Reply:
I have a preference to not declare top level constants and just inline values. If we need to use variables its ideal that they are declared nearby where they are used so that it doesn't involve a lot of context switching to figure out what the variable values are